readme.md

```diff
@@ -1,3 +1,37 @@
-# Author's Haven
+# Author's Haven - A Social platform for the creative at heart.
 
-A Social platform for the creative at heart.
+# Getting started
+
+This project contains frontend implementation for the Author's Haven project. If you are interested in checking out the backend, please do follow this link:
+[persephone-ah-backend](https://github.com/andela/persephone-ah-backend)
+
+Start with the following instructions
+
+Clone repo to your local machine and chnage directory into the folder
+
```

Comment:
Hi @devPinheiro, good job with the readme file, I noticed a typo error, 'chnage' I think it should be 'change'

Reply:
I have fixed this. Thanks for pointing it out.